backend/src/main/java/com/morak/back/appointment/domain/AvailableTimes.java

```diff
@@ -0,0 +1,35 @@
+package com.morak.back.appointment.domain;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.persistence.CascadeType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class AvailableTimes {
+
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "appointment_id", nullable = false, updatable = false)
+    private Set<AvailableTime> availableTimes = new HashSet<>();
+
+    public boolean hasMember(Long memberId) {
+        return this.availableTimes.stream()
+                .anyMatch(availableTime -> availableTime.matchMember(memberId));
+    }
+
+    public void select(Set<LocalDateTime> localDateTimes, Long memberId) {
+        final List<AvailableTime> availableTimes = localDateTimes.stream()
+                .map(dateTime -> AvailableTime.builder().memberId(memberId).startDateTime(dateTime).build())
```

Comment:
```suggestion
                .map(dateTime -> new AvailableTime(memberId, dateTime)
```

c: 이제 보니까 꼭 빌더를 고집할 필요는 없을 것 같은데, 어떤가요 ?

Reply:
저도 사실 바꾸고 싶었는데요. 수정해야할 부분이 많을 것 같아서 안바꿨답니다.
다들 바꾸는거 찬성하시나요? 

찬성 - 👍
반대 - 👎

Reply:
기존 빌더쓰는 부분을 지울필요는 없을것같고, 생성자와 빌더를 둘다 열어두면 되지않을까요?

Reply:
인정

Reply:
둘다 유연하게 가져가도록 하죠~ 너무 불편했었슴다